otter_welcome_buddy/common/utils/image.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from discord import Member
+from discord import User
+from PIL import Image
+from PIL import ImageDraw
+from PIL import ImageFont
+from PIL.Image import Image as ImageType
+from PIL.ImageDraw import ImageDraw as ImageDrawType
+from PIL.ImageFont import FreeTypeFont
+
+from otter_welcome_buddy.common.constants import FONT_PATH
+
+_DEFAULT_IMAGE_PATH: str = os.path.dirname(os.path.realpath(__file__)) + "/image.jpg"
+_COLOR_TEXT: str = "black"
+_COLOR_OUTLINE: str = "gray"
+_COLOR_BACKGROUND: str = "white"
```

Comment:
All these vars may be better on a config file, so then we can make a config component to read it from instead of leaving them in the global scope, WDT?

Reply:
Unless we're doing something else with them I don't think is worth it, I would prefer to keep them here for now but if some moment we add another functionality where we use them again we can add them to the config file.